### ISSUE TITLE
Any plan to release training code?

### ISSUE THREAD

**Issue by facetohard** [jefferyZhan/Griffon#14]

**Comment by jefferyZhan** [comment]
Yes. In a few days, we'll release the Griffon-G version and then the training codes.

**Comment by facetohard** [comment]
Amazing!

**Comment by jefferyZhan** [comment]
Hi @facetohard ! We have released the Griffon-G checkpoints and updated the inference codes. Welcome to try our new model.

**Comment by facetohard** [comment]
@jefferyZhan I have tried your excellent work and I want to follow your work and do some extand works, Any plan to release training code?

**Comment by junedgar** [comment]
@jefferyZhan Good job!  When will you release the training code? Looking forward to it.

**Comment by jefferyZhan** [comment]
Thank you for your interest in our Griffon. We may release the training code after the CVPR rebuttal.

**Comment by jefferyZhan** [comment]
@junedgar Thank you for your interest in our Griffon. We may release the training code after the Spring Festival.

**Comment by xiazhi1** [comment]
> Very soon. We have finished converting the experimental codes into the griffon packages and are retraining using the released version for double-check. After verification, we'll merge them into the main branch. Thank you for your continuous attention for our work.很快。我们已经完成了实验代码到 griffon 软件包的转换，并正在使用已发布的版本进行重新训练和复核。验证完成后，我们会将其合并到主分支。感谢您一直以来对我们工作的关注。

@jefferyZhan Good work. Will you release the training code and ablation experiment code of Griffon-G? Especially the ablation experiment of replacing the vision encoder with dinov2, I am very interested in it.

**Comment by jefferyZhan** [comment]
@xiazhi1 We'll release the full training codes and include the dinov2 encoder code as requested. Thank you for your interest in our work.

**Comment by xiazhi1** [comment]
> [@xiazhi1](https://github.com/xiazhi1) We'll release the full training codes and include the dinov2 encoder code as requested. Thank you for your interest in our work.

Excellent！ Could you please tell me how to replace the vision encoder with clip in the training code? What codebase is it based on? I want to try to replace the pre-trained MLLM vision encoder with other encoders for training. Can you share your successful experience? If it is convenient, can we add WeChat to discuss it separately? Any suggestions are welcome！

**Comment by RobinHan24** [comment]
> [@xiazhi1](https://github.com/xiazhi1) We'll release the full training codes and include the dinov2 encoder code as requested. Thank you for your interest in our work.

so,when will you release the training code? Thank you very much.

**Comment by jefferyZhan** [comment]
@RobinHan24  Hi, our code and data are released now. Thanks for your interest in our work.
